src/nni_manager/training_service/pai/paiYarn/paiYarnTrainingService.ts

```diff
@@ -283,10 +281,8 @@ class PAIYarnTrainingService extends PAITrainingService {
                 Authorization: `Bearer ${this.paiToken}`
             }
         };
-        request(submitJobRequest, (error: Error, response: request.Response, body: any) => {
+        request(submitJobRequest, (error: Error, response: request.Response, _body: any) => {
             if ((error !== undefined && error !== null) || response.statusCode >= 400) {
-                const errorMessage: string = (error !== undefined && error !== null) ? error.message :
```

Comment:
@SparkSnail This `errorMessage` is not used?

Reply:
Seems only have declare of `errorMessage`, and `this.log.error(errorMessage)` is missing for some reason. I think it's better to add `this.log.error(errorMessage)` instead of deleting it.